src/main/java/org/broadinstitute/hellbender/utils/downsampling/PositionalDownsampler.java

```diff
@@ -71,13 +71,16 @@ private void handlePositionalChange( final GATKRead newRead ) {
         // Use ReadCoordinateComparator to determine whether we've moved to a new start position.
         // ReadCoordinateComparator will correctly distinguish between purely unmapped reads and unmapped reads that
         // are assigned a nominal position.
-        if ( previousRead != null && ReadCoordinateComparator.compareCoordinates(previousRead, newRead, header) != 0 ) {
-            if ( reservoir.hasFinalizedItems() ) {
```

Comment:
Why has this `if` check gone?

Reply:
Because we want to prime the downsampler for the first group. This is necessary to keep it truly deterministic instead of having the first location in each shard using the default seed, as it turns out the reservoir itself has defensive checks against finalizing when its empty so the defensiveness here doesn't save us much.